ReactAndroid/src/main/java/com/facebook/react/modules/network/ForwardingCookieHandler.java

```diff
@@ -138,25 +138,9 @@ protected void doInBackgroundGuarded(Void... params) {
         // https://bugs.chromium.org/p/chromium/issues/detail?id=559720
         return null;
       } catch (Exception exception) {
-        String message = exception.getMessage();
-        // We cannot catch MissingWebViewPackageException as it is in a private / system API
-        // class. This validates the exception's message to ensure we are only handling this
-        // specific exception.
-        // The exception class doesn't always contain the correct name as it depends on the OEM
-        // and OS version. It is better to check the message for clues regarding the exception
-        // as that is somewhat consistent across OEMs.
-        // For instance, the Exception thrown on OxygenOS 11 is a RuntimeException but the message
-        // contains the required strings.
-        // https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/webkit/WebViewFactory.java#348
-        if (exception.getClass().getCanonicalName().contains("MissingWebViewPackageException")
-            || (message != null
-                && (message.contains("WebView provider")
-                    || message.contains("No WebView installed")
-                    || message.contains("Cannot load WebView")))) {
-          return null;
-        } else {
-          throw exception;
-        }
+        // fatal exception is no good for the user experience, 
```

Comment:
I'm generally against this change as we would blindly catch all the exceptions and return `null` here while we want to distinguish between `MissingWebViewPackageException` and other type of exceptions

Reply:
@cortinico i totally agree. this is not a best practice, or in principal good code. the thought behind this although here is that, in any case when the `CookieManager` instance is not available, return a `null` is better than the `fatal` exception. 

Probably we can log such framework errors to further research the root cause.